include/powerloader/context.hpp

```diff
@@ -26,6 +26,11 @@ namespace powerloader
 
     namespace details
     {
+        // Even if this is a private function, it is called form a template.
```

Comment:
Following that reasoning, then both should be marked `POWERLOADER_API` and there is no need to explain, it's not msvc-specific, the same issue can happen with other compilers and different flags (in particular the optimizing ones). If it doesn't happen on other toolchains it might be that we force exporting all symbols which should be considered an issue as that's the opposite of what we want.

Reply:
It does not happen with the WIndows toolchain on the CI of this repo, but it appeared on the Windows toolchain of conda-forge (the compiler is one build ahead of that of this repo), so I was not sure if that was a bug or not.

I also noticed a weird thing about `mirror_map_type` which is not exported, used by the powerloader executable, and does not produce any link error (neither on this CI nor on conda-forge).

Reply:
Indeed `mirror_map_type` have functions implemented in the CPP and should be marked `POWERLOADER_API` too... I think I might have introduced other ones like that, sorry ^^;